components/atoms/Text/Text.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import {
+  StyleProp,
+  StyleSheet,
+  Text as NativeText,
+  TextStyle,
+} from 'react-native'
+import useTheme from '../../hooks/useTheme'
+
+export interface TextProps {
+  /** Text color from the palette. eg. cta
+   *
+   * default: `cta`
+   */
+  color?: any
```

Comment:
@J-Rehan It doesn't seem right to have a prop for color when the convention for `<Text />` is to pass color in through the styles prop. What's your thinking with this? Is it just because you're trying to default it to neutral? Can't you still set a default through the styles object?

Reply:
Sure, we can do that.